pkg/gofr/datasource/pubsub/nats/nats_embedded.go

```diff
@@ -0,0 +1,30 @@
+package nats
+
+import (
+	"time"
+
+	"github.com/nats-io/nats-server/v2/server"
+)
+
+const embeddedConnTimeout = 10 * time.Second
+
+// RunEmbeddedNATSServer starts a NATS server in embedded mode.
+func RunEmbeddedNATSServer() (*server.Server, error) {
```

Comment:
The `RunEmbeddedNATSServer` seems to be unused, please specify it's purpose ?

Reply:
removed, was used for testing